php/class-photonfill.php

```diff
@@ -507,8 +537,9 @@ public function get_attachment_image( $attachment_id, $size = 'full', $attr ) {
 					$html = $this->get_lazyload_image( $attachment_id, $size, $attr );
 				} else {
 					$alt = ( ! empty( $attr['alt'] ) ) ? ' alt=' . esc_attr( $attr['alt'] ) : '';
+					$style = ( ! empty( $attr['style'] ) ) ? ' style=' . esc_attr( $attr['style'] ) : '';
 					$classes = $this->get_image_classes( ( empty( $attr['class'] ) ? array() : $attr['class'] ), $attachment_id, $size );
-					$html = '<img sizes="' . esc_attr( $this->get_responsive_image_attribute( $attachment_id, $size, 'sizes' ) ) . '" srcset="' . esc_attr( $this->get_responsive_image_attribute( $attachment_id, $size, 'srcset' ) ) . '" class="' . esc_attr( $classes ) . '" ' . $alt . '>';
+					$html = '<img sizes="' . esc_attr( $this->get_responsive_image_attribute( $attachment_id, $size, 'sizes' ) ) . '" srcset="' . esc_attr( $this->get_responsive_image_attribute( $attachment_id, $size, 'srcset' ) ) . '" class="' . esc_attr( $classes ) . '" ' . $alt . ' ' . $style . '>';
```

Comment:
This is tough to read due to the concatenation (same with line 565 and 632, and maybe others in this file). I would use `sprintf`, which would make the string significantly more palatable:
```php
'<img sizes="%s" srcset="%s" class="%s" %s %s>'
```